corio.py

```diff
@@ -260,9 +230,7 @@ def main(options):
                                                 terminated_tests=test_ids)
                 schedule.cancel_job(sched_job)
                 if options.support_bundle:
-                    resp = collect_upload_sb_to_nfs_server(MOUNT_DIR, sb_identifier,
-                                                           max_sb=CORIO_CFG["max_no_of_sb"])
-                    LOGGER.info("collect support bundles response: %s", resp)
+                    support_bundle_process(CORIO_CFG["sb_interval_mins"] * 60, sb_identifier)
```

Comment:
Please revert this change since we don't want to collect support bundle in loop again. A process started at line 183 should already be collecting the support bundle in given interval. Here we just want to collect single support bundle due to failure.